ui/selectors/multichain.ts

```diff
@@ -132,53 +126,25 @@ export function getMultichainNetwork(
 
   if (isEvm) {
     // EVM networks
-    const evmChainId: string = getCurrentChainId(state);
+    const evmChainId: Hex = getCurrentChainId(state);
 
     // TODO: Update to use network configurations when @metamask/network-controller is updated to 20.0.0
     // ProviderConfig will be deprecated to use NetworkConfigurations
     // When a user updates a network name its only updated in the NetworkConfigurations.
     const evmNetwork: ProviderConfigWithImageUrlAndExplorerUrl =
       getProviderConfig(state);
-    // Fallback to a known network image if network configuration does not defined it
+
```

Comment:
Despite the above comment about deprecating `providerConfig`, I'm not sure its possible to do it here yet.  This seems like a multichain wrapper around  `providerConfig`.  And so would need to stay this way as long as we're still shimming `getProviderConfig`